<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageView</class>
 <widget class="QWidget" name="MessageView">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="TextBrowser" name="textBrowser">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="openLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SearchEditor" name="searchEditor"/>
   </item>
   <item>
    <widget class="QLabel" name="helpLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LineEditor" name="lineEditor"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchEditor</class>
   <extends>QLineEdit</extends>
   <header>searcheditor.h</header>
  </customwidget>
  <customwidget>
   <class>TextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>textbrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineEditor</class>
   <extends>QLineEdit</extends>
   <header>lineeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
